pkg/plugins/runtime/k8s/controllers/pod_converter.go

```diff
@@ -44,6 +44,7 @@ func (p *PodConverter) PodToDataplane(
 	services []*kube_core.Service,
 	others []*mesh_k8s.Dataplane,
 ) error {
+	logger := converterLog.WithValues("Dataplne.name", dataplane.Name, "Pod.name", pod.Name)
```

Comment:
```suggestion
	logger := converterLog.WithValues("Dataplane.name", dataplane.Name, "Pod.name", pod.Name)
```

Reply:
It seems unusual to me to put `.` in log keys. @jakubdyszkiewicz opinions?

Reply:
I wanted to preserve the same format as for ZoneIngress and ZoneEgress https://github.com/kumahq/kuma/blob/master/pkg/plugins/runtime/k8s/controllers/pod_converter.go#L66

Reply:
yeah, I'd try to avoid `.`